types/spotify-web-playback-sdk/index.d.ts

```diff
@@ -33,7 +34,9 @@ declare namespace Spotify {
     type ErrorTypes = 'account_error' | 'authentication_error' | 'initialization_error' | 'playback_error';
 
     interface Image {
+        height: number;
         url: string;
+        width: number;
```

Comment:
As per https://developer.spotify.com/documentation/web-api/reference/object-model/#image-object, `height` and `width` may be `null` or `undefined`, so this should be `height?: number | null` and `width?: number | null`.

Reply:
good point!